### ISSUE TITLE
If pages not exist in other languages

### ISSUE THREAD

**Issue by frase-git** [cmb69/polyglott_xh#14]
Beispiel:
2 Sprachen: Englisch und Deutsch
Im Englischen gibt es eine Seite "Test".
Im Deutschen gibt es diese Seite **noch** nicht.
Ziel ist es aber (in den meisten Fällen), in beiden Sprachen gleichen Content zu haben.
Im Plugin auf der Seite "Übersetzungen" gibt es keinerlei Hinweise auf Seiten, die es in der aktuellen Sprache **noch nicht** gibt. Sie taucht gar nicht auf.
Das kann bei umfangreicher Seitenanzahl zu Problemen führen.
Wunsch:
Auf "Übersetzungen" sollten (evtl. unten) Seiten aufgelistet sein, die es in anderen Sprachen gibt - in der aktuellen aber nicht.
Vielleicht mit einem Ausrufezeichen (FA) - nicht Warndreieck - weil das ja auch beabsichtigt sein könnte.

**Comment by cmb69** [comment]
> Im Plugin auf der Seite "Übersetzungen" gibt es keinerlei Hinweise auf Seiten, die es in der aktuellen Sprache noch nicht gibt. Sie taucht gar nicht auf.

Da sehe ich eigentlich kein besonderes Problem. Polyglott_XH ist so gedacht, dass man eine Hauptsprache hat, in der Seiten immer zuerst erstellt werden, und in dieser Sprache werden dann auch alle Seiten angezeigt. Vielleicht sollte die entsprechende Übersicht in Zweitsprachen gar nicht angezeigt, und statt dessen auf die Übersicht in der Hauptsprache verlinkt werden?

**Comment by frase-git** [comment]
> Vielleicht sollte die entsprechende Übersicht in Zweitsprachen gar nicht angezeigt, und statt dessen auf die Übersicht in der Hauptsprache verlinkt werden?

Ja, aber was ist, wenn es eine Seite in der **Zweit**sprache gibt und in der Hauptsprache noch nicht?

Und überhaupt: wechselt man im Adminmodus bei einer Plugin-Seite die Sprache, landet man prompt auf einer **leeren** Seite. Auch nicht schön.
Mir fiel das alles auf, bei der Arbeit an xh.org.

**Comment by cmb69** [comment]
> Ja, aber was ist, wenn es eine Seite in der Zweitsprache gibt und in der Hauptsprache noch nicht?

Da würde ich sagen: Fehler des Nutzers. Ist ein bisschen wie eine Übersetzung eines Textes, die den Text ändert, weil der Übersetzer meint es besser zu wissen. Das ist ein Wartungsalbtraum mit dem ich mich leider gelegentlich rumärgern muss. 

> Und überhaupt: wechselt man im Adminmodus bei einer Plugin-Seite die Sprache, landet man prompt auf einer leeren Seite. Auch nicht schön.

Nein, nicht schön, aber schwierig zu lösen; ich meine mich dunkel zu erinnern, das besser machen gewollt zu haben, aber irgendwelche Probleme haben mich dann zur jetzigen Lösung bewogen, nämlich dass man immer auf der Startseite (ohne Query-String) landet, wenn eine Seite nicht übersetzt ist.

> Mir fiel das alles auf, bei der Arbeit an xh.org.

Ah! Dort sollte dann vielleicht wirklich die Hauptsprache auf `de` umgestellt werden, und `en` zur Zweitsprache degradiert werden. Zumindest derzeit fehlt es leider an Leuten, die die englische Version warten würden.
